src/test/identity-utils.ts

```diff
@@ -14,69 +14,98 @@ describe('ID Translation Tests', () => {
 
     let testBlockstackId3 = 'ankit.something_crux.foo'
 
+    let testBlockstackDomain = 'exodus_crux'
+    let testCruxDomainId = 'exodus.crux'
+    let testBlockstackId = 'exodus_crux.id'
+
+    let testBlockstackId4 = 'foobar.id'
+    let testInvalidNamespaceId = 'alice.foo'
+    let testCruxInvalidNamespaceId = 'alice@foo.bar'
+
+    it('blockstack id initialization without subdomain', () => {
+        let bsid = BlockstackId.fromString(testBlockstackId4);
+        expect(bsid).to.be.instanceOf(BlockstackId);
+        expect(bsid.toString()).to.be.eql(testBlockstackId4);
+    })
+
+    it('BlockstackId.fromString should fail with BlockstackIdInvalidStructure', () => {
+        expect(() => BlockstackId.fromString("alice")).to.throw(PackageError).with.property('errorCode', PackageErrorCode.BlockstackIdInvalidStructure);
+    })
 
     it('blockstack id translates to crux id', () => {
         let bsid = BlockstackId.fromString(testBlockstackId1)
         expect(IdTranslator.blockstackToCrux(bsid).toString()).to.equal(testCruxId1)
     })
 
+    it('blockstack domain id translates to crux domain id', () => {
+        const bsDomainId = new BlockstackDomainId(testBlockstackDomain);
+        const cruxDomainId = IdTranslator.blockstackToCrux(bsDomainId);
+        expect(cruxDomainId).to.be.instanceOf(CruxDomainId);
+        expect(IdTranslator.blockstackToCrux(bsDomainId).toString()).to.equal(testCruxDomainId);
```

Comment:
shouldn't it be :

expect(cruxDomainId.toString()).to.equal(testCruxDomainId);

Reply:
Fixed